nullaway/src/main/java/com/uber/nullaway/ErrorMessage.java

```diff
@@ -24,12 +24,21 @@
 
 /** Contains error message string to be displayed and the message type from {@link MessageTypes}. */
 public class ErrorMessage {
+
+  static final String COVERED = "(Covered) ";
+
   MessageTypes messageType;
   String message;
+  public final boolean covered;
 
-  public ErrorMessage(MessageTypes messageType, String message) {
+  public ErrorMessage(MessageTypes messageType, String message, boolean covered) {
```

Comment:
We probably want to document this method and give that last argument a better name than `covered`. Something like `autofixCandidate`? Or even `autofixable`?

Reply:
Would you please tell me if ```fixable``` might work too ?